lerna.json

```diff
@@ -0,0 +1,6 @@
+{
+  "npmClient": "yarn",
+  "packages": ["packages/*"],
+  "version": "0.20.0",
+  "useWorkspaces": true
```

Comment:
nit-nit: may suggest to put this option near to `npmClient`, as those are closely related